docs/source/guide/FAQ.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Troubleshoot Label Studio
+title: Troubleshoot Labeling Issues
```

Comment:
Is this the right title for us to be using? These all aren't just labeling issues, but can be issues with Label Studio.

Reply:
Ah - you're right. It should be sentence case. Habit of mine. I'll fix this (I think there are a few others too.)

Reply:
I wanted to move this out of Getting Started, because it seems to technical for a gentle introduction to the product. From what I can tell, the sections most out of place are:

**Blank page when loading a project** -- this should be in a "project troubleshooting" page, but we we don't have one. We can keep it here until I create one. 
**HTML label offsets are in the wrong places** -- this would be better in "Import/Export troubleshooting." Again, I can create that in a separate PR. I already found another issue to add to that one.

Reply:
I'm concerned about consistent terminology. For example, meta_title reads "Troubleshoot Label Studio". Also share @hogepodge 's concern - the scope of troubleshooting in this document goes beyond labeling. If this is one step in a broader refactor of the documentation, I think this is ok.

Reply:
@amymrou - Good catch on the meta title. I'll fix that. 

My next PR is going to be surfacing our troubleshooting info in the docs. We currently have 7 troubleshooting sections. Five of them are subsections sprinkled throughout the docs. Two are their own pages. I want to consolidate in some places and make them all top-level pages. 

From this page, there are two issues that are not labeling related. They'll "live" here briefly, but I'll move them out in my next PR where I'll be focusing on troubleshooting.